### PR TITLE
[1.1.1.1] Tweak DNS over Discord docs

### PR DESCRIPTION
**Pull request by MattIPv4**
Small cleanups to the docs for DNS over Discord, see commit messages for exact changes.

cc https://github.com/MattIPv4/DNS-over-Discord/pull/56

### CODE REVIEW

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ title: DNS over Discord
 
 1.1.1.1 works from a Discord server, thanks to the 1.1.1.1 bot. [Invite the bot to your Discord server](https://cfl.re/3nM6VfQ) to start using DNS over Discord.
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 2]
```diff
@@ -19,6 +20,7 @@ Example:
 /dig domain: cloudflare.com
 ```
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 3]
```diff
@@ -58,31 +60,33 @@ Discord has a limit of 25 options in slash commands, so DNS over Discord offers
 
 To query other DNS record types, or multiple record types at once, use the `/multi-dig` command.
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 4]
```diff
@@ -58,31 +60,33 @@ Discord has a limit of 25 options in slash commands, so DNS over Discord offers
...
 
-When you request a response in the short form, the name and TTL columns will be excluded. The command only returns the data column without formatting, similar to the equivalent `dig` command response.
+When you request a response in the short form, the name and TTL columns will be excluded. The command only returns the data column without formatting, similar to the equivalent `dig` command-line interface response.
 
 Example:
 
 ```txt
-/dig domain: cloudflare.com type: AAAA short: true
+/dig domain: cloudflare.com type: AAAA records short: True
 ```
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 5]
```diff
@@ -58,31 +60,33 @@ Discord has a limit of 25 options in slash commands, so DNS over Discord offers
...
-The refresh button is available on all responses to the `dig` command, including those that resulted in an error, such as an unknown domain or no records found.
+The refresh button is available on all responses to the `/dig` command, including those that resulted in an error, such as an unknown domain or no records found.
 
 <div class="medium-img">
 
 ![Refreshing dig domain: cloudflare.com](/1.1.1.1/static/dns-over-discord/dig-command-refresh.gif)
 
 </div>
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 6]
```diff
@@ -93,6 +97,7 @@ Example:
 /multi-dig domain: cloudflare.com types: A AAAA
 ```
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 7]
```diff
@@ -152,28 +157,31 @@ Use a \* (asterisk) in place of a record type to get DNS results for all support
 </div>
 </details>
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 8]
```diff
@@ -152,28 +157,31 @@ Use a \* (asterisk) in place of a record type to get DNS results for all support
...
+Like the main `/dig` command, the `/multi-dig` command also supports the optional short flag after the types have been specified in the slash command.
 
 Example:
 
 ```txt
-/multi-dig domain: cloudflare.com types: CDS CDNSKEY short: true
+/multi-dig domain: cloudflare.com types: CDS CDNSKEY short: True
 ```
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 9]
```diff
@@ -152,28 +157,31 @@ Use a \* (asterisk) in place of a record type to get DNS results for all support
...
-As with the `dig` command, any user can press the refresh button to refresh the displayed DNS results, including for DNS queries that had previously failed.
+As with the `/dig` command, any user can press the refresh button to refresh the displayed DNS results, including for DNS queries that had previously failed.
 
 <div class="medium-img">
 
 ![Refreshing multi-dig domain: cloudflare.com types: A AAAA](/1.1.1.1/static/dns-over-discord/multi-dig-command-refresh.gif)
 
 </div>
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 10]
```diff
@@ -187,21 +195,24 @@ Examples:
 /whois query: 13335
 ```
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 11]
```diff
@@ -187,21 +195,24 @@ Examples:
...
 ## Other commands
 
 The bot also has a set of helper commands available to get more information about the bot and quick links.
 
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 12]
```diff
@@ -187,21 +195,24 @@ Examples:
...
 
-Example: 
+Example:
 
 ```txt
 /help
 ```
 
-### privacy command
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 13]
```diff
@@ -211,7 +222,8 @@ Example:
 /privacy
 ```
 
-### github command
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 14]
```diff
@@ -221,7 +233,8 @@ Example:
 /github
 ```
 
-### invite command
+
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by marciocloudflare** [comment]
Thanks @MattIPv4. I've removed the extra spaces before H3, as per our content strategy - I'll keep an eye for any stragglers that may still linger in the doc :)

**Comment by MattIPv4** [comment]
Tyvm!

**Pull request merged**